etc/helm/pachyderm/values.yaml

```diff
@@ -239,6 +239,51 @@ loki-stack:
     config:
       lokiAddress: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
       snippets:
+        scrapeConfigs: |
+          - job_name: kubernetes-pods
+            pipeline_stages:
+              {{- toYaml .Values.config.snippets.pipelineStages | nindent 4 }}
+              - match:
+                  selector: '{namespace!~"{{ .Release.Namespace }}"}'
```

Comment:
Is this an anchored regexp; i.e. does a release namespace of "a" match a target in the namespace "aa"?  If it's not anchored, we'll need to change it to something like `^{{.Release.Namespace}}$`.

Reply:
Good question. Based on the documentation from https://grafana.com/docs/loki/latest/logql/log_queries/, it looks like the `=~` operator is anchored, but the `!~` is not mentioned:
```
Note: The =~ regex operator is fully anchored, meaning regex must match against the entire string, including newlines. The regex . character does not match newlines by default. If you want the regex dot character to match newlines you can use the single-line flag, like so: (?s)search_term.+ matches search_term\n.
```
I think it is probably better to just use `!=` which should be an exact match

Reply:
By the way, the test procedure you listed above works! Additionally I can port-forward the individual lokis and I don't see pods mixing with logcli

Reply:
That's good to hear.  And yeah, != is better than ~= if it exists.

I did try this:
```
$ kubectl create ns foo.bar
The Namespace "foo.bar" is invalid: metadata.name: Invalid value: "foo.bar": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```
I am pretty sure that no namespace is a regex that can match a different namespace.